client/src/models/trees.js

```diff
@@ -139,14 +139,24 @@ const trees = {
       const data = response.data
       this.receiveTreeCount(data)
 
-      const query =
-        `${
-          process.env.REACT_APP_API_ROOT
-        }/api/${getOrganization()}trees?filter[order]=${orderBy} ${order}&filter[limit]=${rowsPerPage}&filter[skip]=${
-          page * rowsPerPage
-        }&filter[fields][id]=true&filter[fields][timeCreated]=true&filter[fields][status]=true` +
-        `&filter[fields][planterId]=true&filter[fields][treeTags]=true&filter[where][active]=true` +
-        (filter ? filter.getBackloopString() : '')
+      const where = filter ? filter.getWhereObj() : {}
+   
+      const lbFilter = JSON.stringify({
+        where: {...where, active: true},
```

Comment:
so this always filters for active, no matter what's passed in?

Reply:
That's right. We're using `active=true` to mean _not deleted_.
Perhaps this isn't the best place to be setting it though; it might make more sense if it's returned as part of `where` from `filter.getWhereObj()